internal/polls.js

```diff
@@ -0,0 +1,29 @@
+
+module.exports.getConstitutionPoll = (contracts, proposal) => {
+  return contracts.polls.methods.constitutionPolls(proposal).call();
+}
+
+module.exports.getDocumentPoll = (contracts, proposal) => {
+  return contracts.polls.methods.documentPolls(proposal).call();
+}
+
+module.exports.constitutionHasAchievedMajority = (contracts, proposal) => {
+  return contracts.polls.methods.constitutionHasAchievedMajority(proposal).call();
```

Comment:
Why do we need .call() here?

Reply:
That's a web3 thing, it makes the read/call you defined actually happen.